### PR TITLE
Added code to rerun tasks, and test requing etl tasks.

### PR DESCRIPTION
**Pull request by mklauber**

### CODE REVIEW

File: seo/tests/test_admin.py [hunk 1]
```diff
@@ -97,4 +102,41 @@ def test_seo_site_parent_cannot_become_child(self):
         self.assertEqual(resp.status_code, 200)
         self.assertContains(resp, "field-parent_site errors")
         parent_site_refresh = SeoSite.objects.get(pk = parent_seo_site.pk)
-        self.assertEqual(parent_site_refresh.parent_site, None)
\ No newline at end of file
+        self.assertEqual(parent_site_refresh.parent_site, None)
```
Comment (aspidites) [inline seo/tests/test_admin.py]: `assertIsNone`

File: seo/tests/test_admin.py [hunk 2]
```diff
@@ -97,4 +102,41 @@ def test_seo_site_parent_cannot_become_child(self):
...
+        self.date = datetime(year=2015, month=1, day=1, hour=1, minute=1, second=1, tzinfo=pytz.utc)
+        self.bu = BusinessUnit(id=self.id, date_updated=self.date, date_crawled=self.date)
+        self.bu.save()
+        self.etl_to_solr = TaskState(state="test", task_id="testing",
+                                     name="priority_etl_to_solr", tstamp=datetime.now(),
+                                     args="""(u'8f3dcec2-264d-49fb-98df-8c6f52fb3936', u'24974', u"Gold's Gym")""",
+                                     kwargs="{}")
+        self.etl_to_solr.save()
+
+    @override_settings(CELERY_ALWAYS_EAGER=True,
```
Comment (aspidites) [inline seo/tests/test_admin.py]: :+1: much better than the setattr(settings, "foo") stuff we have elsewhere.

**Comment by aspidites** [comment]
Don't know enough about tasks to say if anything is glaringly wrong. Looks fine though.

**Comment by darrint** [comment]
+1

**Pull request merged**